src/tools/illink/src/ILLink.Shared/DiagnosticId.cs

```diff
@@ -182,9 +182,9 @@ public enum DiagnosticId
 		DynamicallyAccessedMembersOnTypeReferencesMemberOnBaseWithDynamicallyAccessedMembers = 2115,
 		RequiresUnreferencedCodeOnStaticConstructor = 2116,
 		MethodsAreAssociatedWithUserMethod = 2117,
-		CompilerGeneratedMemberAccessedViaReflection = 2118,
-		DynamicallyAccessedMembersOnTypeReferencesCompilerGeneratedMember = 2119,
-		DynamicallyAccessedMembersOnTypeReferencesCompilerGeneratedMemberOnBase = 2120,
+		_unused_CompilerGeneratedMemberAccessedViaReflection = 2118,
+		_unused_DynamicallyAccessedMembersOnTypeReferencesCompilerGeneratedMember = 2119,
+		_unused_DynamicallyAccessedMembersOnTypeReferencesCompilerGeneratedMemberOnBase = 2120,
```

Comment:
Will we also doc that people shouldn't expect this warning anymore? We could even go as far as saying this is safe to suppress since it's likely this generates more false positives than point out real issues.